DemoApp/Sources/Views/VideoCallParticipantModifier/DemoVideoCallParticipantModifier.swift

```diff
@@ -34,66 +34,23 @@ struct DemoVideoCallParticipantModifier: ViewModifier {
 
     func body(content: Content) -> some View {
         content
-            .adjustVideoFrame(to: availableFrame.size.width, ratio: ratio)
-            .overlay(
-                ZStack {
-                    BottomView(content: {
-                        HStack {
-                            ParticipantInfoView(
-                                participant: participant,
-                                isPinned: participant.isPinned
-                            )
-                            
-                            if showAllInfo {
-                                Spacer()
-                                ConnectionQualityIndicator(
-                                    connectionQuality: participant.connectionQuality
-                                )
-                            }
-                        }
-                        .padding(.bottom, 2)
-                    })
-                    .padding(.all, showAllInfo ? 16 : 8)
-                    
-                    if participant.isSpeaking && participantCount > 1 {
-                        Rectangle()
-                            .strokeBorder(Color.blue.opacity(0.7), lineWidth: 2)
-                    }
-                    
-                    if popoverShown {
-                        ParticipantPopoverView(
-                            participant: participant,
-                            call: call,
-                            popoverShown: $popoverShown
-                        ) {
-                            PopoverButton(
```

Comment:
Now we don't have stats anywhere? Would be good to keep them here, until the new design is ready.